lib/trailblazer/finder.rb

```diff
@@ -5,31 +5,31 @@
 require "dry-types"
 require "ostruct"
 
-require "trailblazer/finder/utils/array"
-require "trailblazer/finder/utils/hash"
-require "trailblazer/finder/utils/string"
-require "trailblazer/finder/utils/splitter"
-require "trailblazer/finder/utils/extra"
-require "trailblazer/finder/adapters/active_record/predicates"
-require "trailblazer/finder/adapters/active_record/paging"
-require "trailblazer/finder/adapters/active_record/sorting"
-require "trailblazer/finder/adapters/kaminari/paging"
-require "trailblazer/finder/adapters/will_paginate/paging"
-require "trailblazer/finder/adapters/sequel/predicates"
-require "trailblazer/finder/adapters/sequel/paging"
-require "trailblazer/finder/adapters/sequel/sorting"
-require "trailblazer/finder/adapters/basic/predicates"
-require "trailblazer/finder/adapters/basic/paging"
-require "trailblazer/finder/adapters/basic/sorting"
-require "trailblazer/finder/activities/prepare"
-require "trailblazer/finder/activities/process"
-require "trailblazer/finder/activities/find"
-require "trailblazer/finder/helpers/basic"
-require "trailblazer/finder/helpers/sorting"
-require "trailblazer/finder/dsl"
-require "trailblazer/finder/find"
-require "trailblazer/finder/base"
-require "trailblazer/operation/finder"
+require_relative "finder/utils/array"
+require_relative "finder/utils/hash"
+require_relative "finder/utils/string"
+require_relative "finder/utils/splitter"
+require_relative "finder/utils/extra"
+require_relative "finder/adapters/active_record/predicates"
```

Comment:
What's the benefit of `require_relative` here, I wonder? :coffee:

Reply:
I had a issue where $LOAD_PATH was affected by another gem, and this code broke. 

Not a bug , but since the files are located in the same folder,  i changed it.

I'm testing the whole stack with new version of the gems.